src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -139,6 +139,9 @@
     public static final String STRAND_BIAS_BY_SAMPLE_KEY =          "SB";
     public final static String TRANSMISSION_PROBABILITY_KEY =       "TP"; //PhaseByTransmission
     public static final String ZERO_COVERAGE_LOCI =                 "ZL"; //DiagnoseTargets
+    public static final String UNCONVERTED_BASE_COVERAGE_KEY =       "UNCONVERTED_BASE_COV";
+    public static final String CONVERTED_BASE_COVERAGE_KEY =         "CONVERTED_BASE_COV";
+    public static final String METHYLATOION_CONTEXT_KEY =            "CONTEXT";
```

Comment:
This one is misspelled... `METHYLATOION_CONTEXT_KEY`

Reply:
Also, "CONTEXT" is pretty generic, I would suggest making it something more specific so its recognizable when seen in the context of a VCF.